bluetooth-music/app.js

```diff
@@ -138,6 +139,14 @@ function handleUrl (url) {
       }
       uploadEvent(protocol.AUDIO_STATE.PLAYING)
       break
+    case '/STOPPED':
```

Comment:
This is inconsistent with others.

Reply:
This is private URL, how to separate it from public URL?

Reply:
Conventionally, we would name the url with prefix `_` and keep the rest of the name consistent with other urls.

Reply:
> Conventionally, we would name the url with prefix `_` and keep the rest of the name consistent with other urls. Or made the prefix explicit, e.g. `/private/stopped` or `/event/stopped`.

Good instruction.

Reply:
Why do we have the private URLs?

Reply:
Due to 

> Why do we have the private URLs?

2 reasons:
 - When bluetooth-music app is not launched, user can play music in phone, so the /PLAYING event can only be received by bluetooth-service, so we need forward this event to music-app as private URL.
 - In this bug, when directly close bluetooth when playing, there's only `close` event while no `stop` event, so we need send a mock `STOPPED` event to music-app by private URL.